modules/lang-painless/src/main/java/org/elasticsearch/painless/action/PainlessExecuteAction.java

```diff
@@ -284,15 +284,8 @@ static Request parse(XContentParser parser) throws IOException {
         Request(StreamInput in) throws IOException {
             super(in);
             script = new Script(in);
-            if (in.getVersion().before(Version.V_6_4_0)) {
```

Comment:
Do we need to keep the 6.4 back compat code here?

Reply:
Good catch. Will change back.